dashboard/src/shared/ClusterServiceClass.ts

```diff
@@ -1,4 +1,4 @@
-import { axios } from "./Auth";
+import { axiosWithAuth } from "./AxiosInstance";
```

Comment:
couple of options here to avoid having to change this everywhere:

1. Keep axiosWithAuth in ./Auth and import it as `{ axios as axiosWithAuth }` in the index
2. Import this as `{ axiosWithAuth as axios }` in all these imports

That said, I like that `axiosWithAuth` is fairly descriptive, so I honestly don't have a strong opinion here.

Reply:
I didn't know about that notation but that may obscure a bit the origin of the `axios` instance that it's using and can be potentially confused for the one without the auth header. I'd leave this as it is to make clear the difference.

Reply:
Agreed, sounds good to me.